packages/block-library/src/comments-pagination-numbers/edit.js

```diff
@@ -0,0 +1,22 @@
+/**
+ * WordPress dependencies
+ */
+import { useBlockProps } from '@wordpress/block-editor';
+
+const PaginationItem = ( { content, tag: Tag = 'a', extraClass = '' } ) => (
+	<Tag className={ `page-numbers ${ extraClass }` }>{ content }</Tag>
+);
+
+export default function QueryPaginationNumbersEdit() {
```

Comment:
For consistency with other places we could rename it to `CommentsPaginationNumbersEdit`.